deploy/docs/Best_Practices.md

```diff
@@ -1095,3 +1096,39 @@ The reliability of the system can be calculated using the following formulas:
 
 - If limited by queue_size: `number_of_instances*timeout[min]*sending_queue.queue_size/load_in_DPM` minutes.
 - If limited by PVC size: `number_of_instances*PVC_size/(1KB*load_in_DPM)` minutes.
+
+## Assigning Pod to particular Node
```

Comment:
I'd maybe split that into 2 levels: 1 for using `nodeSelector` and 1, lower level, for describing how to use it to pin pods to linux nodes. WDYT?